target/linux/ramips/image/mt7621.mk

```diff
@@ -891,6 +891,24 @@ define Device/wevo_w2914ns-v2
 endef
 TARGET_DEVICES += wevo_w2914ns-v2
 
+define Device/xiaomi_mi-router-ac2100
+  $(Device/uimage-lzma-loader)
+  BLOCKSIZE := 128k
+  PAGESIZE := 2048
+  KERNEL_SIZE := 4096k
+  IMAGE_SIZE := 124416k
+  UBINIZE_OPTS := -E 5
+  IMAGES += kernel1.bin rootfs0.bin
+  IMAGE/kernel1.bin := append-kernel
+  IMAGE/rootfs0.bin := append-ubi | check-size
+  IMAGE/sysupgrade.bin := sysupgrade-tar | append-metadata
+  DEVICE_VENDOR := Xiaomi
+  DEVICE_MODEL := Mi Router AC2100
+  DEVICE_TITLE := Xiaomi Mi Router AC2100 (Black, Cylinder)
```

Comment:
Probably You'll ask why I announced them with their visual definition at menuconfig.
Because there is huge mess in forum topic about white and black editions.
I though this will be better for users which not familier with openwrt and these devices.
And will stop further mess in github, forum, wiki.

Reply:
In my opinion it would be very beneficial to have these names in menuconfig. I was already asked several times which Router model is which... If we want to stop hundreds of questions in the forum and here in commit comments or bug reports it would be much better to include it in menuconfig and not only in a wiki. Of course the wiki page on top is a very useful and necessary thing.

Reply:
Why not add that descriptive string to `DEVICE_MODEL` instead of overriding `DEVICE_TITLE` which will only make a difference in menuconfig, but does **not** show up in web-wizards like https://chef.libremesh.org/

Reply:
Because that was inteded by us. We don't want to see this larger title in the WebUI but we want to help the users selecting the right device in menuconfig.

Reply:
Our intention was that it shouldn't show up in the WebUI or somewhere else but just to give an easier selection possibility in menuconfig. If we change DEVICE_MODEL it will also change the names there, right? We still have the shorter model name in the dts file definitions so I think it might make sense to use DEVICE_TITLE. What do you think?

Reply:
Maybe let's continue to discuss this in the review comment of pablomh. It will be confusing to discuss the same thing twice at different places...

Reply:
Lets continue this discussion in here:
https://github.com/openwrt/openwrt/pull/3070#discussion_r433291316